src/signals/incident/components/StepByStepNavClickable/StepByStepNavClickable.test.tsx

```diff
@@ -154,8 +154,13 @@ describe('StepByStepNavClickable when form is valid', () => {
     )
 
     await waitFor(() => {
-      userEvent.type(screen.getByText('step1'), ' ')
+      fireEvent.keyDown(screen.getByText('step1'), {
```

Comment:
normally always prefer userEvent but firerevent only triggers the onkeydown once, what is prefered in this case